static/js/L.CanvasOverlay.js

```diff
@@ -0,0 +1,136 @@
+
+/*
+  Generic  Canvas Overlay for leaflet,
+ Stanislav Sumbera, April , 2014
```

Comment:
Good question. It appears to be unlicensed but there are alternatives that we should be able to drop in such as [the one from CartoDB](https://github.com/CartoDB/Leaflet.CanvasLayer), which is explicitly MIT licensed.